read_tasks.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+todo_task_file=$1
+padding="       "
```

Comment:
This should probably be exported in `todo.sh` to affect all later sourced files.